filters/filter_background_blur.py

```diff
@@ -0,0 +1,32 @@
+import cv2
+import numpy
+import numpy as np
+
+from filters.filter import Filter
+from filters.filter_blur import FilterBlur
+
+
+class FilterBackgroundBlur(FilterBlur):
```

Comment:
FilterBackgroundBlur uses all of the same code as FilterBlur except for this method.